src/chewieConfig.js

```diff
@@ -30,10 +31,19 @@ const config = {
   },
 
   generationResult: {
-    srcLocation: 'git@github.com:hybris/chewie-sample-result.git',
-    branch: 'master',
-    cloneLocation: 'latestResultRepo'
+    srcLocation: process.env.generationResultLocation || 'ssh://git@stash.hybris.com:7999/wookiee/devportal_out.git',
+    branch: process.env.docuBranch || 'dev',
+    cloneLocation: 'latestResultRepo',
+    s3: {
+      bucket: 'devportal-html-content',
```

Comment:
Well, you are right. What about `process.env.bucket` || 'devportal-html-content'`?